### DISCUSSION TITLE
Using postgis with knex

### DISCUSSION THREAD

**Discussion by m-row** [adonisjs/core#3256] [Help] [answered]
Hello, whats the best way to extend knex instance with libraries like: [https://www.npmjs.com/package/knex-postgis](knex-postgis) for adonis?

**Comment by thetutlage** [comment, marked as answer]
Added to the roadmap for current release. I will soon release a cookbook on same https://trello.com/c/y8PCAodY/47-september-planning-2021

**Reply by m-row** [reply]
Great! looking forward to it

**Reply by thetutlage** [reply]
Documented here https://docs.adonisjs.com/cookbooks/using-knex-postgis-with-lucid
